config/initializers/job_configurations.rb

```diff
@@ -54,14 +53,9 @@
         args: -> { [Time.zone.today] },
       },
       # Combined Invoice Supplement Report to S3
-      combined_invoice_supplement_report: {
-        class: 'Reports::CombinedInvoiceSupplementReport',
-        cron: cron_24h,
-        args: -> { [Time.zone.today] },
-      },
       combined_invoice_supplement_report_v2: {
         class: 'Reports::CombinedInvoiceSupplementReportV2',
-        cron: cron_24h_1am,
+        cron: cron_24h,
```

Comment:
the time change was so that we didn't run two giant tables scans simultaneously. now that we have just this report, it should be fine to move the new report earlier